guides/common/modules/proc_adding-images-to-server.adoc

```diff
@@ -61,13 +61,13 @@ Use the `--uuid` field to store the full path of the image location on the KVM s
 ----
 endif::[]
 ifeval::["{context}" == "rhv-provisioning"]
-Use the `--uuid` option to store the template UUID on the {OVirt} server.
+Use the `--uuid` option to store the template UUID on the {oVirt} server.
```

Comment:
These are attributes. Have you checked does this match the attributes.adoc file?

Reply:
Yes: [attributes.adoc line 49](https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes.adoc#L49)